src/utils/hearing.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+
+export function getMainSection(hearing) {
+  return _.find(hearing.sections, (section) => section.type === "main");
+}
+
+/*
+* Return URL to hearing view. Accepts optional fullscreen parameter
+* to force fullscreen query parameter.
+ */
+export function getHearingURL(hearing, {fullscreen} = {}) {
+  const url = `/hearing/${hearing.id}`;
```

Comment:
Cool. Didn't know backend already returns the same response with `/v1/hearing/<slug>` request. Will change. I'll double check that all hearing links also use this function so that we can close #247 as a side effect.

Didn't manage to reproduce the redirection problem you mentioned. `/hearing/slug` does indeed redirect to `hearing/slug?fullscreen=true` when hearing should default to fullscreen.